frontend/src/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -47,6 +49,29 @@ const ModelRegistryRoutes: React.FC = () => (
             />
             <Route path="*" element={<Navigate to="." />} />
           </Route>
+          <Route path="versions/archive">
```

Comment:
I'm leaving that here, not just for your PR, maybe I'll create a task for this, but we should start thinking about splitting routes to decouple a little bit this section.